server/README.md

```diff
@@ -9,16 +9,23 @@
 ## Development
 
 ```
-# copy file and set proper data inside
-cp .env.example .env
+# Environmental variables
+
+Copy `.env.example` to `.env` file and fill environmental variables
+
+-   REDIS_PORT: Redis port (default: 6379)
+-   REDIS_HOST: Redis host (default: 127.0.0.1)
+-   REDIS_PASSWORD: Redis password (default: demo)
+
+# Run docker compose or install redis with RedisJson module manually
```

Comment:
or go to https://redislabs page and choose free redis (something like this)

Reply:
done